src/main/java/com/minecolonies/coremod/inventory/InventoryCitizen.java

```diff
@@ -406,7 +408,10 @@ public void createMaterialStore(@NotNull final MaterialSystem system)
         }
     }
 
-    /**
+    public MaterialStore getMaterialStore()
+    {
+        return materialStore;
+    }    /**
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ATrailingCommentCheck)